app/views/comments/_comment.html.erb

```diff
@@ -1,27 +1,56 @@
 <li class="comment">
-  <p class="author"><strong><%= comment.user.username %></strong></p>
-  <%= comment.body.html_safe %>
-  <% if current_user %>
-    <a class="create-link" href="javascript:void(0)" id="link-collapse-<%= comment.id %>" onclick="collapseReplyDiv('collapse-<%= comment.id %>')" style="font-size: 12px;">Reply</a>
-      <div class="form-comment" id="form-collapse-<%= comment.id %>" style="display: none">
-        <%= form_for :comment, url: comments_path, html: {class: 'new-form' } do |form| %>
-            <p class="form-p">
-              <%= form.label :body, 'Enter comment:' %>
-              <%= form.cktext_area :body, class: 'textarea-field', id: "comment_#{comment.id}", ckeditor: { uiColor: '#ffe6bd', toolbar: 'mini', height: 100, width: 400 } %>
-            </p>
-        
-            <%= form.hidden_field :commentable_type, value: "Comment" %>
-            <%= form.hidden_field :commentable_id, value: comment.id %>
-            <%= form.hidden_field :parent_article, value: @article.id %>
+  <div class="comment-vote">
+    <% if current_user %>
+      <% if comment.votes.where(user: current_user).any? %>
+        <%= up_vote_links("Comment", comment.id, :put) %>
+        <div id="vote-Comment-count-<%= comment.id %>" class="comment-vote-count"><%= comment.votes.sum(:value) %></div>
+        <%= down_vote_links("Comment", comment.id , :put) %>
+      <% else %>
+        <%= up_vote_links("Comment", comment.id , :post) %>
+        <div id="vote-Comment-count-<%= comment.id %>" class="comment-vote-count"><%= comment.votes.sum(:value) %></div>
+        <%= down_vote_links("Comment", comment.id , :post) %>
+      <% end %>
+    <% else %>
+      <%= link_to("", nil, id: "vote-comment-up-nil-#{comment.id}", class: "comment-vote-top") %>
+      <div id="vote-count-<%= comment.id %>" class="comment-vote-count"><%= comment.votes.sum(:value)  %></div>
+      <%= link_to("", nil , id: "vote-comment-down-nil-#{comment.id}", class: "comment-vote-bottom") %> 
+    <% end %>
+    
+    <script type="text/javascript">
```

Comment:
inline javascript is generally frowned upon. What do you think of moving this into a javascript file in your asset pipeline?

You might wrap it into a condition to ensure it only runs on relavent pages:

```js
const $comments = $('#vote-Comment-button-up-*')
if ($comments.length) {
  // your stuff here
}
```

That selector might not work, and if not, then I'd suggest breaking up the long ID, or adding a class to the links,
like `.button-up`